website/pages/chapters/assessing_performance.js

```diff
@@ -230,8 +289,8 @@ export default function AssessingPerformance() {
           So even though value we really care about is the <em>true error</em>,
           we will use the error on the test set as a stand-in for that value. We
           call the error made by the model on the test set the <b>test error</b>
-          , which we can compute. In the case of regression using RSS as the
-          loss function
+          <strike>, which we can compute</strike>. In the case of regression,
```

Comment:
Take out the strike and maybe change it to "a value we <em>can</em> compute"

The point here is to emphasize this is something that is computable in most contexts, unlike the true error.